verification/serverless.yml

```diff
@@ -170,4 +170,17 @@ functions:
               - Authorization
               - X-Api-Key
               - X-Amz-Security-Token
-              - X-Amz-User-Agent
\ No newline at end of file
+              - X-Amz-User-Agent
+
+  get-verification:
+    handler: verification/application/handlers/verification_handlers.get_status
+    role: ${file(./config.${self:provider.stage}.json):ROLE}
```

Comment:
can we make the payload as if it was suppoesed to be hit by api gateway if needed we can use this in future seamlessely.

Reply:
dont need to add API gateway right now.
orchestrator which will call verification to get org_uuid, then after getting org_uuid it will call registry for more details
and orchestator will have authentication